### ISSUE TITLE
Firefox 120 various crashes 

### ISSUE THREAD

**Issue by mcdonc** [NixOS/nixpkgs#269887]
### Describe the bug
Clicking on an image in a Mastodon post causes Firefox to crash under Nix packaging of Firefox 120, pulling up the crashreporter dialog.

### Steps To Reproduce
1. Install firefox 120.
2. Visit https://chattingdarkly.org/@chrism/111440520427357590
3. Click on the image of the christmas tree.

### Expected behavior
The image should be enlarged.  Instead, the crashreporter dialog comes up.

To work around the problem, I reverted to 119.1

### Additional context
Variations on other crashes under Firefox 120 have been reported on Mastodon from other folks, probably related to the same underlying issue.  See https://fosstodon.org/@adingbatponder/111470803473750038

### Notify maintainers
@lovesegfault @hexa

### Metadata
```console
❯ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.62, NixOS, 23.05 (Stoat), 23.05.20231122.8f11807`
 - multi-user?: `yes`
 - sandbox: `relaxed`
 - version: `nix-env (Nix) 2.15.3`
 - channels(root): `"nixos-23.05, unstable"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by teutat3s** [comment]
Probably related to https://github.com/NixOS/nixpkgs/issues/269571

**Comment by teto** [comment]
losely related but I have firefox configured via home-manager and another thing is when you click "restart" on the crashreporter, it doesn't restart.

**Comment by mweinelt** [comment]
This issue should be resolved through #269680.

> another thing is when you click "restart" on the crashreporter, it doesn't restart.

Yeah, I'm aware, but I haven't dug into that yet.
